waspc/src/Wasp/Analyzer/StdTypeDefinitions.hs

```diff
@@ -41,6 +41,10 @@ makeDeclType ''App
 -- | Collection of domain types that are standard for Wasp, that define what the Wasp language looks like.
 -- These are injected this way instead of hardcoding them into the Analyzer in order to make it
 -- easier to modify and maintain the Wasp compiler/language.
+
+-- *** MAKE SURE TO UPDATE: The `validateUniqueDeclarationNames` function in the `Wasp.AppSpec.Valid` module
```

Comment:
This is good, but I normally would put NOTE, because NOTE is so popular that editors will paint it in different color (same like TODO). Up to you though, just a suggestion.

Reply:
I wanted for this comment to stand out more, that's why the custom prefix made sense to me :)